oracles/pyth/anchor/programs/pythexample/src/error.rs

```diff
@@ -0,0 +1,11 @@
+use anchor_lang::prelude::*;
+
+#[error_code]
+pub enum ErrorCode {
+    #[msg("PythError")]
+    PythError,
+    #[msg("TryToSerializePriceAccount")]
```

Comment:
I think maybe make these log messages a bit more descriptive, even if it is a simple example